ci-operator/step-registry/ipi/conf/libvirt/ipi-conf-libvirt-commands.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+touch ${SHARED_DIR}/cluster-config.yaml
+MULTIARCH_RELEASE_IMAGE_INITIAL=registry.ci.openshift.org/ocp-${ARCH}/release-${ARCH}:${BRANCH}
+if [[ -n "${MULTIARCH_RELEASE_IMAGE_INITIAL}" ]]; then
+  echo "Installing from initial release ${MULTIARCH_RELEASE_IMAGE_INITIAL}"
+  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${MULTIARCH_RELEASE_IMAGE_INITIAL}"
+  yq write --inplace ${SHARED_DIR}/cluster-config.yaml OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}
+fi
+
+openshift-install version
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+declare -A SUBNETS
+  SUBNETS["${CLUSTER_TYPE}-0-0"]="126"
```

Comment:
Is this the section that needs to be updated to 1->5

Reply:
Requires the tunnel to be stable before updating.

Reply:
This is just a mapping array, it should be harmless to update independently of tunnel status. It's the lease what determines which `subnet` is chosen.